src/app/components/model_tree/model_tree_line.js

```diff
@@ -25,6 +25,13 @@ angular
             // start = everything up to end
 
             var name = scope.item.name;
+
+            // name *should* always exists but there is a bug where some items don't make it into the catalog.
```

Comment:
Is there an issue for that bug that we could link here so its easy to see if it's finally resolved?